diff --git a/src/brpc/policy/http_rpc_protocol.cpp b/src/brpc/policy/http_rpc_protocol.cpp
index 88b9f83e..5de08989 100644
--- a/src/brpc/policy/http_rpc_protocol.cpp
+++ b/src/brpc/policy/http_rpc_protocol.cpp
@@ -654,12 +654,12 @@ void SerializeHttpRequest(butil::IOBuf* /*not used*/,
 
     Span* span = accessor.span();
     if (span) {
-        hreq.SetHeader("x-b3-traceid", butil::string_printf(
-                           "%016llx", (unsigned long long)span->trace_id()));
-        hreq.SetHeader("x-b3-spanid", butil::string_printf(
-                           "%016llx", (unsigned long long)span->span_id()));
-        hreq.SetHeader("x-b3-parentspanid", butil::string_printf(
-                           "%016llx", (unsigned long long)span->parent_span_id()));
+        hreq.SetHeader("x-bd-trace-id", butil::string_printf(
+                           "%llu", (unsigned long long)span->trace_id()));
+        hreq.SetHeader("x-bd-span-id", butil::string_printf(
+                           "%llu", (unsigned long long)span->span_id()));
+        hreq.SetHeader("x-bd-parent-span-id", butil::string_printf(
+                           "%llu", (unsigned long long)span->parent_span_id()));
     }
 }
 
